### PR TITLE
getOrCreateCarbonSession method parameter is an empty string repair[JIRA CARBONDATA-3119]

### PR DESCRIPTION
**Pull request by zygitup**
1.No interface changes
 
2.Backward compatibility is not affected
 
3.The document does not need to be updated

4.
 val carbon = SparkSession.builder().config(sc.getConf).getOrCreateCarbonSession("")
 val carbon = SparkSession.builder().config(sc.getConf).getOrCreateCarbonSession("  ")
 carbon.sql("SELECT * FROM TABLE").show()   is ok.

### CODE REVIEW

**Comment by CarbonDataQA** [comment]
Can one of the admins verify this patch?

**Comment by xubo245** [comment]
Please optimize the title: [CARBONDATA-3119]Fix the error of getOrCreateCarbonSession method parameter is an empty string

File: format/pom.xml [hunk 1]
```diff
@@ -15,58 +15,59 @@
     See the License for the specific language governing permissions and
     limitations under the License.
 -->
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```
Comment (xubo245) [inline format/pom.xml]: no need to change, please keep the original format

File: integration/presto/Presto_Cluster_Setup_For_Carbondata.md [hunk 2]
```diff
@@ -1,135 +0,0 @@
-# Presto Multinode Cluster setup For Carbondata
```
Comment (xubo245) [inline integration/presto/Presto_Cluster_Setup_For_Carbondata.md]: Why do you remove it?

File: pom.xml [hunk 3]
```diff
@@ -614,7 +614,7 @@
       <modules>
         <module>integration/spark2</module>
         <module>integration/hive</module>
-        <module>integration/presto</module>
+        <!--<module>integration/presto</module>-->
```
Comment (xubo245) [inline pom.xml]: no need to change, please keep the original format

File: pom.xml [hunk 4]
```diff
@@ -556,7 +556,7 @@
...
```
Comment (xubo245) [inline pom.xml]: no need to change, please keep the original format

File: pom.xml [hunk 5]
```diff
@@ -496,7 +496,7 @@
...
```
Comment (xubo245) [inline pom.xml]: no need to change, please keep the original format

File: pom.xml [hunk 6]
```diff
@@ -463,7 +463,7 @@
         <module>integration/spark2</module>
         <module>examples/spark2</module>
         <module>integration/hive</module>
-        <module>integration/presto</module>
+        <!--<module>integration/presto</module>-->
```
Comment (xubo245) [inline pom.xml]: no need to change, please keep the original format

File: integration/spark2/src/main/scala/org/apache/spark/sql/CarbonSession.scala [hunk 7]
```diff
@@ -269,9 +271,9 @@ object CarbonSession {
       session
     }
 
-    /**
-     * It is a hack to get the private field from class.
-     */
+     /**
```
Comment (xubo245) [inline integration/spark2/src/main/scala/org/apache/spark/sql/CarbonSession.scala]: no need to change, please keep the original format

File: integration/spark2/src/main/scala/org/apache/spark/sql/CarbonSession.scala [hunk 8]
```diff
@@ -171,7 +173,7 @@ object CarbonSession {
     }
 
     def getOrCreateCarbonSession(storePath: String,
-        metaStorePath: String): SparkSession = synchronized {
+                                 metaStorePath: String): SparkSession = synchronized {
```
Comment (xubo245) [inline integration/spark2/src/main/scala/org/apache/spark/sql/CarbonSession.scala]: no need to change, please keep the original format

File: integration/spark2/src/main/scala/org/apache/spark/sql/CarbonSession.scala [hunk 9]
```diff
@@ -114,8 +115,8 @@ class CarbonSession(@transient val sc: SparkContext,
   }
 
   private def withProfiler(
-      sqlText: String,
-      generateDF: (QueryExecution, SQLStart) => DataFrame): DataFrame = {
+                            sqlText: String,
```
Comment (xubo245) [inline integration/spark2/src/main/scala/org/apache/spark/sql/CarbonSession.scala]: no need to change, please keep the original format

File: integration/spark2/src/main/scala/org/apache/spark/sql/CarbonSession.scala [hunk 10]
```diff
@@ -80,32 +81,32 @@ class CarbonSession(@transient val sc: SparkContext,
...
   def isDataMapHit(sqlStatement: String, dataMapName: String): Boolean = {
     // explain command will output the dataMap information only if enable.query.statistics = true
     val message = sql(s"EXPLAIN $sqlStatement").collect()
     message(0).getString(0).contains(dataMapName)
   }
 
-  /**
-   * Run SparkSQL directly
-   */
+   /**
```
Comment (xubo245) [inline integration/spark2/src/main/scala/org/apache/spark/sql/CarbonSession.scala]: no need to change, please keep the original format

File: integration/spark2/src/main/scala/org/apache/spark/sql/CarbonSession.scala [hunk 11]
```diff
@@ -80,32 +81,32 @@ class CarbonSession(@transient val sc: SparkContext,
...
+        new Dataset[Row](self, qe, RowEncoder(qe.analyzed.schema))
       }
     )
   }
 
-  /**
-   * Return true if the specified sql statement will hit the datamap
-   * This API is for test purpose only
-   */
+   /**
```
Comment (xubo245) [inline integration/spark2/src/main/scala/org/apache/spark/sql/CarbonSession.scala]: no need to change, please keep the original format

**Pull request closed without merging**